apps/challenges/models.py

```diff
@@ -277,6 +277,8 @@ def __init__(self, *args, **kwargs):
     )
     # Id in the challenge config file. Needed to map the object to the value in the config file while updating through Github
     config_id = models.IntegerField(default=None, blank=True, null=True)
+    # Store the metadata for the default submission meta attributes of this challenge phase.
+    default_submission_meta_attributes = JSONField(default=None, blank=True, null=True)
```

Comment:
`# Store the metadata for the default submission meta attributes of this challenge phase.` -> `# Store the default metadata for a submission meta attributes of a challenge phase.`